content/source/docs/enterprise/sentinel/import/tfrun.html.md

```diff
@@ -0,0 +1,111 @@
+static data regarding the Terraform Cloud [workspace][workspace-glossary] associated with the run.
+
+```
+tfrun
+└── workspace
+```
+
+-> Note: When policies using this import, keep in mind that workspace
+data is generally editable by users outside of the context of policy
+enforcement. For example, consider the case of omitting the enforcement of
+policy rules for development workspaces by the workspace name (allowing the
+policy to pass if the workspace ends in `-dev`). While this is useful for
+extremely granular exceptions, the workspace name could be edited by
+workspace admins, effectively bypassing the policy. In this case, where an
+extremely strict separation of policy managers vs. workspace practitioners is
+required, using [policy sets](/docs/enterprise/sentinel/manage-policies.html)
+to only enforce the policy on non-development workspaces is more appropriate.
+
+[run-glossary]: /docs/glossary.html#run
+[workspace-glossary]: /docs/glossary.html#workspace
+
+## Namespace: Workspace
+
+```
+workspace
+├── name (string)
+├── description (string)
+├── auto_apply (bool)
+├── working_directory (string)
+└── vcs_repo (map of keys)
+```
+
+### Value: `name`
+
+* **Value Type:** String.
+
+The `name` value within the [workspace namespace](#namespace-workspace) contains the current run's workspace name.
+
+As an example, in a workspace named `app-dev-us-east` the following policy would evaluate to `true`:
+
+```
+# Enforces production rules on all non-development workspaces
+
+import "tfrun"
+import "strings"
+
+# (Actual policy logic omitted)
+meets_production_policy = rule { ... }
+
+main = rule {
+    if strings.has_suffix(tfrun.workspace.name, "-dev") {
+        true
+    } else
+        meets_production_policy
+    } 
+}
+```
+
+### Value: `description`
+
+* **Value Type:** String.
+
+The `description` value within the [workspace namespace](#namespace-workspace) contains the current run's workspace description.
+
+This value can be `null`.
+
+### Value: `auto_apply`
+
+* **Value Type:** Boolean.
+
+The `auto_apply` value within the [workspace namespace](#namespace-workspace)
+contains the current run's workspace [auto-apply](/docs/enterprise/workspaces/settings.html#auto-apply-and-manual-apply) setting.
+
+### Value: `working_directory`
+
+* **Value Type:** String.
+
+The `working_directory` value within the [workspace namespace](#namespace-workspace)
+contains the working directory of the current run's workspace.
+
+This value is an empty string (`""`) when no working directory is set.
+
+### Value: `vcs_repo`
+
+* **Value Type:** A string-keyed map of values.
+
+The `vcs_repo` value within the [workspace namespace](#namespace-workspace)
+data associated with a VCS repository connected to the workspace.
+
+Details regarding each attribute can be found in the documentation for the Terraform Cloud [Workspaces API](/docs/enterprise/api/workspaces.html).
+
+```
+vcs_repo (map of keys)
```

Comment:
As per the last comment on namespace tree views.

If indeed we are not documenting this under the assumption that it's going to be prone to future change, we could probably just omit this and keep it as the `(map of keys)` that's documented in the workspace namespace view.

Reply:
Not that it's going to be prone to change; just that I didn't see a point in basically rewriting and having to have two sources of truth for everything that's already in the Workspaces API docs.

Reply:
Alright, I think this is fine. Just as long as the conveyance here is clear that we're not really making a habit of documenting individual namespaces like this and that this is mainly to illustrate the layout of an attribute that's mainly just supposed to be a map, versus static fields.